packages/react-components/src/components/FileUploadCards.test.tsx

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+import React from 'react';
+import { _FileUploadCards, FileUploadCardsProps } from './FileUploadCards';
+import { render, screen } from '@testing-library/react';
+import { registerIcons } from '@fluentui/react';
+
+describe('FileUploadCards should be rendered properly', () => {
+  beforeEach(() => {
+    registerIcons({
+      icons: {
+        cancelfileupload: <></>,
+        genericfile24_svg: <></>
+      }
+    });
+  });
+
+  it('should render the component', async () => {
+    const props = {
+      activeFileUploads: [
+        {
+          id: 'MockId',
+          filename: 'MockFileUpload',
+          progress: 50
+        }
+      ]
+    } as FileUploadCardsProps;
+    renderFileUploadCardWithDefaults(props);
+    expect(await screen.findByText('MockFileUpload')).toBeDefined();
```

Comment:
Do we have a way to verify that progress is correct?

Reply:
The `FileUploadCards` doesn't track progress, The `FileCard` tracks the progress (that I previously wrote test for)